test/test_lyric.py

```diff
@@ -0,0 +1,133 @@
+import unittest
+from typing import Dict
+from tuneflow_py.models.song import Song
+from tuneflow_py.models.lyric import Lyrics, LyricLine, LyricWord
+
+
+class MockLyricLine:
```

Comment:
Since we don't have a param "sentence" when setting up a LyricLine object, a data class like that seems a little bit easier and we'd be able to compare test targets with a static data instance. otherwise, we would pass in str and tokenize to set up the LyriciLine object and compare it against test targets. Or if you would suggest any other solutions